Automattic/jetpack
# Moved the WordPress i18n library dependency to regular dependencies.

In order to have a production build we also need WordPress i18n, which has been moved to devDependency, and needs to be moved back. This change moves it back.




Fixes production builds.

#### Changes proposed in this Pull Request:

* Moved the i18n library dependency declaration from `devDependencies` to `dependencies`.

#### Testing instructions:





* Run `yarn build-production`, observe an error about trying to access a property `static` of `undefined`.
* Use this PR, run the build again, see no errors.

#### Proposed changelog entry for your changes:
* N/A
